### ISSUE TITLE
Update platform to reference http://connectors.airbyte.com/files/registry/v0/[oss|cloud]_registry.json

### ISSUE THREAD

**Issue by bnchrch** [airbytehq/airbyte#25092]
# Overview
We want to update the platform and its config to reference the following paths for the new oss and cloud registry

http://connectors.airbyte.com/files/registry/v0/[oss|cloud]_registry.json

## Prerequisites
Blocked by: https://github.com/airbytehq/airbyte-platform-internal/issues/5791

## Worth noting
For roll out it is planned that these paths resolve to the old connector catalogs at `prod-airbyte-cloud-connector-metadata-service/[oss|cloud]_catalog.json`

Once we are ready to roll out the new catalog we will update the mapping in terraform of the path.

**But that is not part of this ticket**
